docs/examples/types_secret_types.py

```diff
@@ -3,19 +3,27 @@
 class SimpleModel(BaseModel):
     password: SecretStr
     password_bytes: SecretBytes
+    password_idempotent: SecretStr
 
-sm = SimpleModel(password='IAmSensitive', password_bytes=b'IAmSensitiveBytes')
+sm = SimpleModel(password='IAmSensitive', 
+                 password_bytes=b'IAmSensitiveBytes', 
```

Comment:
I think this is not required anymore, but I don't like this format, better to do it the way black suggests if you have to.

Reply:
This is fixed now.